expat/Makefile.am

```diff
@@ -115,11 +205,88 @@ buildlib:
 	@echo 'ERROR:' >&2
 	@false
 
+#page
+#### building xmlwf
 
-.PHONY: run-benchmark
-run-benchmark:
-	$(MAKE) -C tests/benchmark
-	./run.sh tests/benchmark/benchmark@EXEEXT@ -n $(top_srcdir)/../testdata/largefiles/recset.xml 65535 3
+bin_PROGRAMS = xmlwf/xmlwf
+
+xmlwf_xmlwf_LDADD = libexpat.la
+xmlwf_xmlwf_SOURCES = \
+    xmlwf/xmlwf.c \
+    xmlwf/xmlfile.c \
+    xmlwf/codepage.c \
+    xmlwf/@FILEMAP@.c
+
+xmlwf_xmlwf_CPPFLAGS = -I$(srcdir)/lib
+
+if MINGW
+if UNICODE
+xmlwf_xmlwf_CPPFLAGS += -mwindows
+xmlwf_xmlwf_LDFLAGS = -municode
+endif
+endif
+
+EXTRA_DIST += \
+    xmlwf/codepage.h \
+    xmlwf/ct.c \
+    xmlwf/filemap.h \
+    xmlwf/readfilemap.c \
+    xmlwf/unixfilemap.c \
+    xmlwf/win32filemap.c \
+    xmlwf/xmlfile.h \
+    xmlwf/xmlmime.c \
+    xmlwf/xmlmime.h \
+    xmlwf/xmltchar.h \
+    xmlwf/xmlurl.h \
+    xmlwf/xmlwin32url.cxx
+
+#page
+#### examples
+
+noinst_PROGRAMS = examples/elements examples/outline
+
+examples_elements_SOURCES = examples/elements.c
+examples_elements_LDADD = libexpat.la
+
+examples_outline_SOURCES = examples/outline.c
+examples_outline_LDADD = libexpat.la
+
+#page
+#### tests
+
+AM_TESTS_ENVIRONMENT = export EXPAT_SRCDIR=$(srcdir); export EXPAT_BUILDDIR=$(builddir);
+noinst_LIBRARIES = libruntests.a
```

Comment:
No more `test/` prefix for `libruntests.a`?

Reply:
Will restore it.